rotkehlchen/tests/integration/test_zksynclite.py

```diff
@@ -341,3 +346,59 @@ def test_decode_swap(zksync_lite_manager, inquirer):  # pylint: disable=unused-a
         notes='Swap fee of 0.1659 DAI',
         address=address,
     )]
+
+
+def test_get_db_transactions(zksync_lite_manager: 'ZksyncLiteManager'):
+    """Test that all zksync lite transactions are loaded from the database."""
```

Comment:
You need to rename the test and mention in the docstring this is a regression test for the issue and point to the issue with a link.

This is not about getting transactions but decoding and showing them.